language/move-vm/runtime/src/unit_tests/vm_arguments_tests.rs

```diff
@@ -166,19 +166,22 @@ impl RemoteCache for RemoteStore {
     }
 }
 
-fn call_script(script: Vec<u8>, args: Vec<Value>) -> VMResult<()> {
+fn call_script_(
```

Comment:
maybe `call_script_and_specify_all_args` or something. Don't care too much since its for tests
Eventually this should just replace `call_script`